packages/drivers/replay-driver/src/replayDocumentServiceFactory.ts

```diff
@@ -5,11 +5,25 @@
 
 import { IDocumentService, IDocumentServiceFactory, IResolvedUrl } from "@fluidframework/driver-definitions";
 import { ISummaryTree } from "@fluidframework/protocol-definitions";
-import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+import { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/common-definitions";
 import { ReplayController } from "./replayController";
 import { ReplayControllerStatic } from "./replayDocumentDeltaConnection";
 import { ReplayDocumentService } from "./replayDocumentService";
 
+/**
+ * Wrapper logger that adds isReplay: true to each event.
+ * ReplayDriver is used in testing/debugging scenarios, so we want to be able to filter these events out sometimes
+ */
+class ReplayDriverTelemetryLogger implements ITelemetryBaseLogger {
```

Comment:
I do not see a reason for this class to exists - can you please use this instead?
```
ChildLogger.create(logger, undefined, { isReplay: true }) 
```

Reply:
Yep just noticed that.  Not sure why my comment didn't appear for you...